### ISSUE TITLE
[Crash] Rare Summoning Crash

### ISSUE THREAD

**Issue by Lycanite** [Lycanite/LycanitesMobs#71]
I still need to reproduce this bug myself, it seems to be very rare, I'll need to do a lot of summoning when testing to try and produce this.

---- Minecraft Crash Report ----
// Daisy, daisy...
Time: 4/5/15 9:09 PM
Description: Unexpected error
java.lang.NullPointerException: Unexpected error
at net.minecraft.client.network.NetHandlerPlayClient.func_147281_a(NetHandlerPlayClient.java:744)
at net.minecraft.network.play.server.S0FPacketSpawnMob.func_148833_a(SourceFile:97)
at net.minecraft.network.play.server.S0FPacketSpawnMob.func_148833_a(SourceFile:15)
at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
at net.minecraft.client.multiplayer.PlayerControllerMP.func_78765_e(PlayerControllerMP.java:273)
at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1590)
at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:961)
at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:887)
at net.minecraft.client.main.Main.main(SourceFile:148)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

**Comment by FrozenPulse** [comment]
I experienced a summon crash myself earlier today. I will try to put other crash logs on here I encounter, if it helps. My single-player game crashed because of a java.util.HashMap exception. Here is the pastebin link for the server crash log:

http://pastebin.com/ae29x6kQ

I was on a default-color Strider in a Mesa Plateau biome. I am using BiomesOPlenty for worldgen. The mobs I was summoning were Gekens -- about four if I remember correctly, using a golden summoning staff. The world seed is 3601737.

I tried reproducing the bug in a different biome in another world, still on a Strider and summoning gekens. The issue has not occurred again.
